src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,27 @@
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleList } from '../PeopleList';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeoplePage: React.FC<Props> = ({ people }) => {
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {people.length === 0 && (
+            <Loader />
+          )}
+
+          <PeopleList
+            people={people}
+          />
```

Comment:
If your component has some designer pieces, then with an empty array they will still be rendered, although there will be no data for this

```suggestion
          {people.length === 0 ? (
            <Loader />
          ) : (
            <PeopleList people={people} />
          )}
```